### ISSUE TITLE
Error when filter result from outer join more than 2 table

### ISSUE THREAD

**Issue by DLT1412** [crate/crate#5469]
**CrateDB version**: 1.0.5

**JVM version**: 1.8.0_92

**OS version / environment description**: CentOS 7

**Problem description**:
Error when filter result after outer join more than two table.

```
SQLActionException[SQLParseException: Couldn't create executionContexts from
NodeOperations: {2=NodeOp{ phase{id=2/nested-loop, nodes=[88GNmORSQ-myfx6d0D6APw], dist=BROADCAST}, downstreamNodes=[88GNmORSQ-myfx6d0D6APw], downstreamPhase=5, downstreamInputId=0}, 0=NodeOp{ phase{id=0/collect, nodes=[88GNmORSQ-myfx6d0D6APw], dist=BROADCAST}, downstreamNodes=[88GNmORSQ-myfx6d0D6APw], downstreamPhase=2, downstreamInputId=0}, 1=NodeOp{ phase{id=1/collect, nodes=[88GNmORSQ-myfx6d0D6APw], dist=SAME_NODE}, downstreamNodes=[88GNmORSQ-myfx6d0D6APw], downstreamPhase=2, downstreamInputId=1}, 6=NodeOp{ phase{id=6/fetchPhase, nodes=[88GNmORSQ-myfx6d0D6APw, Xjr6Sx8UTK-6_c7B7bSTtA]}, downstreamNodes=[], downstreamPhase=2147483647, downstreamInputId=0}}
Leafs: [NestedLoopPhase{executionPhaseId=5, name=nested-loop, joinType=LEFT, joinCondition=op_=(InputColumn{index=2, type=integer},InputColumn{index=6, type=integer}), outputTypes=[integer, integer, string, integer, string, integer, string], jobId=0d998bac-dd27-48b0-9f20-665a3fd10d19, executionNodes=[88GNmORSQ-myfx6d0D6APw]}]
target-sources: {2=[0, 1], 5=[2]}
original-error: Can't handle Symbol RELCOL(join.join.a.b.c, 4)]
```
**Steps to reproduce**:
```
 CREATE TABLE IF NOT EXISTS "db"."order" ( 
 "order_id" STRING ,
 "customer_id" INT, 
 "company_id" INT 
 ) 
 CLUSTERED INTO 1 SHARDS;

  CREATE TABLE IF NOT EXISTS "db"."mt_customer" ( 
 "customer_id" INT ,
 "customer_name" STRING 
 ) 
 CLUSTERED INTO 1 SHARDS;


  CREATE TABLE IF NOT EXISTS "db"."mt_company" ( 
 "company_id" INT ,
 "company_name" STRING 
 ) 
 CLUSTERED INTO 1 SHARDS;

insert into "db.order" ("order_id", "customer_id", "company_id") values ('ORDER#1', 1, 1);
insert into "db.mt_customer" ("customer_id", "customer_name") values (1,'customer_1');
insert into "db.mt_company" ("company_id", "company_name") values (1,'company_1');

```
Query:
[FAILED]
`
select * from ("db.order" a left join "db.mt_customer" b on a.customer_id = b.customer_id) left join "db.mt_company" c on a.company_id = c.company_id where b.customer_id = '1';
`
[SUCCESS]
`
select * from "db.order" a left join "db.mt_customer" b on a.customer_id = b.customer_id where b.customer_id = '1';
`

**Comment by mfussenegger** [comment]
Thanks for reproting, we'll take a look at this.

**Comment by liujie20087349** [comment]
this is also what we need

**Comment by matriv** [comment]
The issue has been fixed and will be available with the upcoming 1.1.3 stable release.

**Comment by bram96** [comment]
I'm trying the queries of DLT1412 in Crate 1.2.0 and got the same error. Isn't this fixed in 1.2.0 yet?
